lxd/daemon.go

```diff
@@ -1012,11 +1013,21 @@ func (d *Daemon) Init() error {
 	// cleanup any state we produced so far. Errors happening here will be
 	// ignored.
 	if err != nil {
+		_, err2 := shared.RunCommandContext(d.shutdownCtx, "systemd-notify", "ERRNO=125") // 125=ECANCELED code
```

Comment:
I did some more digging and it seems that systemd has generic exit codes too: https://www.freedesktop.org/software/systemd/man/latest/systemd.exec.html#Process%20Exit%20Codes

I propose simply exiting with `1` to signal a generic failure.